### PR TITLE
HPCC-14846 Document Multipart Packagemaps use

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@afishbeck Please review

### CODE REVIEW

File: docs/RoxieReference/RoxieRefMods/Packages.xml [hunk 1]
```diff
@@ -303,17 +303,163 @@
...
+&lt;/RoxiePackages&gt;
+</programlisting></para>
+
+    <para>This allows you to add them separately:</para>
+
+    <programlisting>ecl packagemap add-part &lt;target&gt; target_pmid contacts.pkg
+ecl packagemap add-part &lt;target&gt; target_pmid persons.pkg
+</programlisting>
+
+    <para>And then activate the entire packagemap (which consists of all part
```
Comment (afishbeck) [inline docs/RoxieReference/RoxieRefMods/Packages.xml]: "all parts"

**Comment by afishbeck** [comment]
@JamesDeFabia One missing plural, otherwise looks good.

**Comment by JamesDeFabia** [comment]
@afishbeck Please have another look, edits made as requested

**Comment by afishbeck** [comment]
@JamesDeFabia looks good

**Pull request merged**